README.md

```diff
@@ -1,11 +1,44 @@
-# cryoCARE
```

Comment:
We should fix the README to reflect that everything is now in one repo and that we are collaborating on this.

Reply:
Done